ios/FluentIcons/Classes/Image+FluentIcon.swift

```diff
@@ -3,6 +3,29 @@
 //  Licensed under the MIT license.
 //
 
+#if os(macOS)
+import AppKit
+
+private class FluentIconsBundleCheck {}
+
+public extension NSImage {
+  @objc static let fluentIconBundle = Bundle(for: FluentIconsBundleCheck.self)
+
+  @objc static func fluentIcon(_ fluent: FluentIcon) -> NSImage {
+    // Force unwrap here because the resource strings
+    // are generated so we can be confident that the image
+    // exits at runtime.
+    #if SWIFT_PACKAGE
```

Comment:
I think we can remove this check for the mac target since the code looks the same for both options.